examples/protocol_specification_ex/default.yaml

```diff
@@ -24,3 +24,12 @@ ct:ErrorCode: |
     }
   ErrorCodeEnum error_code = 1;
 ...
+---
+entry: [bytes, error]
```

Comment:
I have added dialogue entry and exit points now for the purpose of our discussion. Exit can be both determined by the reply structure AND the message content, hence the reply structure alone is not sufficient to derive exit. I.e. here, the user might receive a bytes message and label the dialogue as `successful` based on its content.

Reply:
It's now much cleaner. Only I would propose different names for these:

`initial-messages` and `final-messages` (or `terminating-messages` or `terminal-messages`)
or
`initiation` and `termination`